src/views/EditorView/Editor/Editor.tsx

```diff
@@ -125,13 +129,29 @@ class Editor extends React.Component<IProps, {}> {
 
     public render() {
         return (
-            <div className="Editor">
-                <canvas
-                    className="ImageCanvas"
-                    ref={ref => EditorModel.canvas = ref}
-                    draggable={false}
-                    onContextMenu={(event: React.MouseEvent<HTMLCanvasElement>) => event.preventDefault()}
-                />
+            <div
+                className="Editor"
+                ref={ref => EditorModel.editor = ref}
+                draggable={false}
+            >
+                <Scrollbars
+                    ref={ref => EditorModel.viewPortScrollbars = ref}
+                    renderTrackHorizontal={props => <div {...props} className="track-horizontal"/>}
```

Comment:
no właśnie to chyba spowodowało że scrollów nie widzę po zbudowaniu appki? tak ma być?/

Reply:
To już opisałem wczoraj wieczorem, będę się starał to jakoś ogarnąć potem, bo niestety by default jest tam cursor pointer wiec musze to jakoś obsłużyć.

Reply:
oki